src/app/clients/ClientsPage.cy.tsx

```diff
@@ -0,0 +1,256 @@
+        it("Fields are set correctly", () => {
+            const expandedClientDetails = rawDataToExpandedClientDetails(
+                sampleRawExpandedClientDetails
+            );
+
+            expect(expandedClientDetails).to.deep.equal({
+                "voucher_#": "VOUCHER_1",
+                full_name: "CLIENT NAME",
+                phone_number: "PHONE NUMBER",
+                packing_date: "04/08/2023",
+                packing_time: new Date("2023-08-04T08:30:00+00:00").getTimezoneOffset()
+                    ? "14:30"
+                    : "13:30",
+                delivery_instructions: "INSTRUCTIONS FOR DELIVERY",
+                address: "Address Line 1, Address Line 2, TOWN, SW1A 2AA",
+                household: "Family of 3 Occupants (2 adults, 1 child)",
+                "age_&_gender_of_children": "5-year-old male",
+                dietary_requirements: "Gluten Free, Halal, No Pasta",
+                feminine_products: "Tampons, Incontinence Pads",
+                baby_products: true,
+                pet_food: "Cat, Dog",
+                other_requirements: "Garlic, Chillies, Hot Water Bottles",
+                extra_information: "EXTRA CLIENT INFORMATION",
+            });
+        });
+
+        it("formatDatetimeAsTime()", () => {
+            expect(formatDatetimeAsTime("2023-08-04T13:30:02+00:00")).to.eq(
+                new Date("2023-08-04T13:30:02+00:00").getTimezoneOffset() === -60
+                    ? "14:30"
+                    : "13:30"
+            );
+            expect(formatDatetimeAsTime("2024-11-23T01:03:50+00:00")).to.eq(
+                new Date("2024-11-23T01:03:50+00:00").getTimezoneOffset() === -60
+                    ? "02:03"
+                    : "01:03"
+            );
+            expect(formatDatetimeAsTime("Invalid_Time_Format")).to.eq("-");
+            expect(formatDatetimeAsTime(null)).to.eq("-");
+        });
+
+        it("formatAddressFromClientDetails()", () => {
+            expect(
+                formatAddressFromClientDetails({
+                    address_1: "Address Line 1",
+                    address_2: "Address Line 2",
+                    address_town: "TOWN",
+                    address_county: "COUNTY",
+                    address_postcode: "POSTCODE",
+                })
+            ).to.eq("Address Line 1, Address Line 2, TOWN, COUNTY, POSTCODE");
+
+            expect(
+                formatAddressFromClientDetails({
+                    address_1: "Address Line 1",
+                    address_2: "",
+                    address_town: "TOWN",
+                    address_county: "",
+                    address_postcode: "POSTCODE",
+                })
+            ).to.eq("Address Line 1, TOWN, POSTCODE");
+        });
+
+        it("formatHouseholdFromFamilyDetails()", () => {
+            expect(
+                formatHouseholdFromFamilyDetails([
+                    { age: 36, gender: "female" },
```

Comment:
fyi age is for children only atm, so 36 is 'invalid data', but I'm indifferent for test data :)